### PR TITLE
lstrojny/fxmlrpc version, pathing changes, and adding configurations

### PR DESCRIPTION
**Pull request by fillmorejd**
Fixing composer version of lstrojny/fxmlrpc

Changing pathing to work outside of root directory of virtual host.  Because of our setup we run this in a subfolder and not at the root of the virtual host.  The new changes will allow the application to do both.

Adding configurations for application.  This allow for a more customizable application.

Let me know if these changes are good for the application. 

Thanks for creating this application!

### CODE REVIEW

File: app/config.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use RomaricDrigon\MetaYaml\Loader\YamlLoader;
+use RomaricDrigon\MetaYaml\MetaYaml;
+use Proton\Application;
+
+class config {
```
Comment (sagikazarmark) [inline app/config.php]: CS: camelCase please.

File: app/config.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    static function setupConfig(Application &$app) {
```
Comment (sagikazarmark) [inline app/config.php]: Objects does not need to be passed by reference.

File: app/config_schema.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+root:
+  _type: prototype
+  _prototype:
+    _type: text
```
Comment (sagikazarmark) [inline app/config_schema.yml]: Missing newline

**Comment by sagikazarmark** [comment]
Nice job @fillmorejd. Thank you very much.

I am willing to merge this for now, but in the future I will try to share more love with this project, which will probably mean huge refactoring.

**Comment by fillmorejd** [comment]
I made the changes you suggested.  Let me know if you have any other suggestions.   

As you move forward let me know if you need any help.

**Comment by sagikazarmark** [comment]
Thanks @fillmorejd 

Will keep you posted as soon as there is a roadmap.

**Pull request merged**